playwright-e2e/pages/pancan/enrollment/consent-form-page.ts

```diff
@@ -22,9 +22,12 @@ export default class ConsentFormPage extends PancanPage {
    * Type: Radiobutton
    * @returns {Promise<void>}
    */
-  async bloodSamples(): Promise<void> {
-    const bloodSamplesRadioButton = new Question(this.page, { classAttr: PatientsData[this.typePatient].ddpTestID.bloodSamples });
-    await bloodSamplesRadioButton.check('Yes', { exactMatch: true });
+  bloodSamples(): Question {
+    return new Question(this.page, { cssClassAttribute: `.${PatientsData[this.typePatient].ddpTestID.bloodSamples}` });
+  }
+
+  async agreeToBloodSamples(answer = 'Yes'): Promise<void> {
+    await this.bloodSamples().check(answer, { exactMatch: true });
```

Comment:
Broke up old `bloodSamples()` function to Question getter and action functions so that Question getter function can be called to make a screenshot in test.